docs/user/reference/features.rst

```diff
@@ -6,4 +6,6 @@ Features
 .. toctree::
    :maxdepth: 1
 
+   ../features
    analytics
+   ../feature-flags
```

Comment:
This is a bit wonky. Probably could use a comment explaining why these are in the directory above:

```suggestion
   ../features
   analytics
   ../feature-flags

.. These are not moved into `reference/features` yet to avoid renames. 
```

Reply:
Added a good bit of commenting here :)